src/definitions/ping1d.json

```diff
@@ -404,7 +404,7 @@
                             "datatype": "u8",
                             "size": "dynamic"
                         },
-                        "description": "An array of return strength measurements taken at regular intervals across the scan region."
+                        "description": "An array of return strength measurements taken at regular intervals across the scan region. The first element represents the closest point to the sensor and the last point the fastest point from the scanning range."
```

Comment:
```suggestion
                        "description": "An array of return strength measurements taken at regular intervals across the scan region. The first element is the closest measurement to the sensor, and the last element is the farthest measurement in the scanned range."
```
Changed "point" to "measurement" because each intensity measurement is over a region in space.

Same for the ping360 ones :-)

Reply:
done.